### PR TITLE
use new gadget properties in gtplus gadgets

### PR DESCRIPTION
**Pull request by gtdeveloper5**
Update gtpluse gadgets to use new gadget property scheme

### CODE REVIEW

File: gadgets/gtPlus/GtPlusAccumulatorWorkOrderTriggerGadget.h [hunk 1]
```diff
@@ -58,6 +58,65 @@ class EXPORTGTPLUSGADGET GtPlusAccumulatorWorkOrderTriggerGadget : public Gadget
 
     /// parameters to control the triggering
 
+    GADGET_PROPERTY_NO_FORCE(image_series, int, "Image series number", 100);
+    GADGET_PROPERTY(verboseMode, bool, "Whether to print more information", false);
+    GADGET_PROPERTY_NO_FORCE(timeStampResolution, float, "Time tick resolution in second", 0.0025f);
```
Comment (hansenms) [inline gadgets/gtPlus/GtPlusAccumulatorWorkOrderTriggerGadget.h]: Why is this a NO_FORCE

File: gadgets/gtPlus/GtPlusAccumulatorWorkOrderTriggerGadget.h [hunk 2]
```diff
@@ -58,6 +58,65 @@ class EXPORTGTPLUSGADGET GtPlusAccumulatorWorkOrderTriggerGadget : public Gadget
 
     /// parameters to control the triggering
 
+    GADGET_PROPERTY_NO_FORCE(image_series, int, "Image series number", 100);
```
Comment (hansenms) [inline gadgets/gtPlus/GtPlusAccumulatorWorkOrderTriggerGadget.h]: Why is this a NO_FORCE?

**Comment by hansenms** [comment]
All the properties declared with NO_FORCE should not be. The NO_FORCE macro is a temporary thing that allows us to mix and match declared and not declared properties. Once all Gadgets have been converted it will be removed. It should only be used in the Gadget base class.

**Comment by hansenms** [comment]
@gtdeveloper5, what is the story on this?

**Comment by hansenms** [comment]
@gtdeveloper5, what is going on with this.

**Comment by gtdeveloper5** [comment]
Was trapped into the previous PR. Will get this one close tomorrow.

**Comment by gtdeveloper5** [comment]
a) This PR is updated to remove the usage of GADGET_PROPERTY_NO_FORCE
b) The cloud node loading from xml files are removed to fit better for the new gadget property scheme. The cloud bus or cloud node file is used to define the cloud nodes. 
c) If cloud bus cannot find any nodes, the node file will be used, instead of running all jobs on the gateway.

**Pull request merged**